SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -2284,6 +2353,42 @@ void ActorSystem::UpdateInterestComponent(AActor* Actor)
 	NetDriver->Connection->SendComponentUpdate(EntityId, &Update);
 }
 
+void ActorSystem::UpdateClientInterest(AActor* Actor, const Worker_EntityId ControllerEntityId, const bool bOverwrite)
+{
+	if (!ensure(GetDefault<USpatialGDKSettings>()->bUseClientEntityInterestQueries))
+	{
+		return;
+	}
+
+	if (APlayerController* PlayerController = Cast<APlayerController>(Actor))
+	{
+		SpatialGDK::ChangeInterestRequest Request;
```

Comment:
minor: could be moved in if scope

Reply:
It's a param in CreateClientInterestDiff